env0/resource_environment_test.go

```diff
@@ -1531,6 +1531,63 @@ func TestUnitEnvironmentWithoutTemplateResource(t *testing.T) {
 		})
 	})
 
+	t.Run("Revision conflict", func(t *testing.T) {
+		createEnvironmentResourceConfigWithRevision := func(environment client.Environment, template client.Template, revision string) string {
+			return fmt.Sprintf(`
+			resource "%s" "%s" {
+				name = "%s"
+				revision = "%s"
+				project_id = "%s"
+				workspace = "%s"
+				terragrunt_working_directory = "%s"
+				force_destroy = true
+				vcs_commands_alias = "%s"
+				without_template_settings {
+					repository = "%s"
+					terraform_version = "%s"
+					type = "%s"
+					revision = "%s"
```

Comment:
IMO preferably remove this "revision" from the test
I think it's more important that this test fails when there's no `revision` in the `without_template_settings` at all